hack/update-deps.sh

```diff
@@ -27,5 +27,6 @@ dep ensure
 
```

Comment:
Since you're here, please also delete OWNERS files under vendor/. 

```rm -rf $(find vendor/ -name 'OWNERS')```

Reply:
Done.